app/views/helpers/pageHeadersAndError.scala.html

```diff
@@ -19,25 +19,26 @@
 @if(backLink.isDefined){
 <a id="backLinkHref" class="link-back" href='@backLink'>@Messages("ated.back")</a>
 @errorDisplay
-<header class="page-header">
-  <h1 id=@headerId class="heading-xlarge">@headerValue</h1>
+    <header class="page-header">
   <p id=@subHeaderId class="heading-secondary"><span class="visuallyhidden">
     @if(subHeaderPrefix.isDefined) {
       @subHeaderPrefix.getOrElse("")
   } else {
     @Messages("ated.screen-reader.section")
   } </span>@subHeaderValue</p>
+    <h1 id=@headerId class="heading-xlarge">@headerValue</h1>
 </header>
 } else {
 <a id="backLinkHref" class="link-back" href='@backLink'>@Messages("ated.back")</a>
 @errorDisplay
 <header class="page-header page-header-margin">
-  <h1 id=@headerId class="heading-xlarge">@headerValue</h1>
+
   <p id=@subHeaderId class="heading-secondary"><span class="visuallyhidden">
     @if(subHeaderPrefix.isDefined) {
```

Comment:
And this can be  

@subHeaderPrefix.getOrElse(@Messages("ated.screen-reader.section"))